backend/de.metas.report/metasfresh-report-service/src/main/java/de/metas/report/server/AbstractReportEngine.java

```diff
@@ -66,12 +69,26 @@ protected ClassLoader createReportClassLoader(final ReportContext reportContext)
 			parentClassLoader = contextClassLoader;
 		}
 
+		
 		final OrgId adOrgId = reportContext.getOrgId();
 		final JasperClassLoader jasperLoader = new JasperClassLoader(adOrgId, parentClassLoader);
+		setPrintFormatIdIfExists(reportContext, jasperLoader);
 		logger.debug("Created jasper loader: {}", jasperLoader);
 		return jasperLoader;
 	}
 
+	private void setPrintFormatIdIfExists(final ReportContext reportContext, final JasperClassLoader jasperLoader)
```

Comment:
rename it to
PrintFormatId getPrintFormatIdOrNull(final ReportContext reportContext)